resources/lang/en/api.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+	'messages'	=> [
+		'email_voucher_history' => '[xXx]Thanks. If you don\'t receive an email with your voucher history, please try again later.'
```

Comment:
Great - why don't you axe the x and merge in master so you can un-do the previous PR Steph did with copy.